examples/ImageStreamer/create_server_profile_based_on_server_profile_template.yml

```diff
@@ -0,0 +1,44 @@
+###
+# Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+- hosts: all
+  vars:
+    config: '{{ playbook_dir }}/oneview_config.json'
+    server_hardware_name: "2-USE722C99W-M, bay 12"
+    server_profile_template: "Windows2016_i3s"
+    profile_name: "profile_i3s"
+
+  tasks:
+
+    - name : "Create Server Profile - {{ profile_name }}"
+      oneview_server_profile:
+        config: "{{ config }}"
+        data:
+          server_hardware: "{{ server_hardware_name }}"
+          server_template: "{{ server_profile_template }}"
```

Comment:
This will not work anymore as `server_template` was not following the style guide from other fields. It should be `serverProfileTemplateName` now.

Reply:
Fixed.